sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -146,7 +147,7 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     try {
       sql(s"CREATE DATABASE $dbName")
       val db1 = catalog.getDatabaseMetadata(dbName)
-      val expectedLocation = makeQualifiedPath(s"spark-warehouse/$dbName.db")
+      val expectedLocation = new Path(makeQualifiedPath(s"spark-warehouse/$dbName.db")).toUri
```

Comment:
I just implement this function with a option[String], I think for a String without a option, it is already simple, we still to implement one?